services/mediastoredata/src/it/java/software/amazon/awssdk/services/mediastoredata/RequestCompressionStreamingIntegrationTest.java

```diff
@@ -63,7 +63,7 @@ public class RequestCompressionStreamingIntegrationTest extends MediaStoreDataIn
 
     @BeforeAll
     public static void setup() {
-        uri = URI.create(createContainer(CONTAINER_NAME).endpoint());
+        uri = URI.create(createContainerIfNotExistAndGetEndpoint(CONTAINER_NAME));
```

Comment:
We probably just want to create one container per account though, otherwise we'll end up having tons of stale containers... We can probably follow the same approach used in https://github.com/aws/aws-sdk-java-v2/blob/master/services/s3control/src/it/java/software.amazon.awssdk.services.s3control/S3MrapIntegrationTest.java#L109, like making container name "java-sdk-test-mediastoredata-compression" + accountId

Reply:
Updated to one container for both tests, per account